RiotSwiftUI/Modules/Template/SimpleUserProfileExample/View/TemplateUserProfile.swift

```diff
@@ -0,0 +1,68 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct TemplateUserProfile: View {
```

Comment:
It can be interesting if we can have pragma marks like we have in view controller templates

Reply:
Any chance you could provide an example? Not sure I fully follow.

Reply:
Oh actually maybe I do, let me know if what I do is correct 😂